### ISSUE TITLE
Potential API shutdown

### ISSUE THREAD

**Issue by mafanikio** [LevPasha/Instagram-API-python#739]
The main question is: Is this API a Legacy API or not?
IG should close the Legacy API at the nearest time and the nearest date could be yesterday. 
Today they announced this post-https://www.instagram.com/developer/
I'm afraid that this API will stop working in April who has any info about it?

**Comment by DeF3c** [comment]
Not your answer but this api acutally work for you?

**Comment by mafanikio** [comment]
Yes

**Comment by LivC182** [comment]
anybody found a workaround ?

**Comment by Exordio** [comment]
Hello everyone, any ideas? its very actual

**Comment by DeF3c** [comment]
Pretty sure this API dont uses any api from instagram

**Comment by Exordio** [comment]
Something nothing already works for me, "challenge required "

**Comment by Drew-Alleman** [comment]
This API should not shutdown as its Unofficial and doesn't use Instagram API, but instead PHP requests!

**Comment by WannaLearnTheRealOne** [comment]
yes, but they said: "Remaining endpoints on the Legacy API will no longer be available after this date and your app's users may lose functionality."
the API uses PHP requests, but the endpoints are controlled by Instagram, if they decide to migrate those endpoints, they will break all the apps using this API.
